rosdep/python.yaml

```diff
@@ -269,6 +269,13 @@ pika:
     pip:
       packages: [pika]
   ubuntu: [python-pika]
+pydantic:
```

Comment:
I think that according to the naming conventions this should be `python-pydantic-pip`

Reply:
yea, you're probably right. I've renamed it.